core/src/main/java/org/infinispan/statetransfer/OutboundTransferTask.java

```diff
@@ -285,8 +286,8 @@ public String toString() {
       return "OutboundTransferTask{" +
             "topologyId=" + topologyId +
             ", destination=" + destination +
-            ", segments=" + segments +
-            ", stateTransferChunkSize=" + stateTransferChunkSize +
+            ", segments=" + new IntSet(segments) +
```

Comment:
well, I guess it would be better to add another ```toStr``` to ```org.infinispan.commons.util.Util``` for the integer set than pay the cost of creating an ```IntSet``` and iterate over the ```Set``` twice...